requirements/requirements-optionals.txt

```diff
@@ -1,8 +1,8 @@
 # Optional packages which may be used with REST framework.
-psycopg2-binary==2.7.5
+psycopg2-binary==2.7.7
 markdown==2.6.11
 django-guardian==1.5.0
-django-filter==1.1.0
+django-filter==2.1.0
```

Comment:
Look at the build... this version does not exist.

Reply:
Issue there is Django Filter dropped PY2 for 2.0. 

Two options: 

1. Skip Django Filter for PY2, similar to 
2. Delay updating here until we drop PY2 ourselves. (Which, I'll confer with the management about, but should be 3.10.) 

*: I basically thinking one or two more 3.9.x releases and then a 3.10 ≈April. 

I think Option 2 is probably easiest: bar dropping PY2, there isn't a significant difference that's relevant to DRF in updating Django Filter.

Reply:
+1 for option 2

Reply:
OK!!